framework/resource/resource.go

```diff
@@ -239,6 +278,60 @@ func (r *Resource) mapperForCreate(cs *createState) (*argmapper.Func, error) {
 	}, argmapper.FuncOnce())
 }
 
+// mapperForStatus returns an argmapper func that will call the resources'
+// defined status function.
+func (r *Resource) mapperForStatus(_ []string) (*argmapper.Func, error) {
```

Comment:
Why does mapperForStatus take this unused arg?

Reply:
Good catch thanks; early in development it did take things but I ended up refactoring them out after I marked it used, and forgot to remove it. Fixed in 5a51eed